weave/trace_server/opentelemetry/python_spans.py

```diff
@@ -278,7 +279,15 @@ def to_call(
         )
         op_name = self.name
         if len(op_name) >= MAX_OP_NAME_LENGTH:
-            op_name = shorten_name(op_name, MAX_OP_NAME_LENGTH)
+            # Since op_name will typically be what is displayed, we don't want to just truncate
+            # Create an identifier abbreviation so similar long names can be distinguished
+            identifier = str(uuid.uuid4())[:4]
```

Comment:
you do not want this to be random - you want it to be deterministic based on the content